recipes/mlgenotype/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "mlgenotype" %}
+{% set version = "0.1.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  #url: "https://test-files.pythonhosted.org/packages/40/e4/483d6d723f3727abbd96404009b773408cebe8d6a0c6478a6f43e9f4d88f/mlgenotype-0.1.12.tar.gz"
+  sha256: e89eb045be093204671c6110a341b4b1dd8ae78c41ba1b1e59b3b4a4c8cabc76
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - rfmodelpredict=mlgenotype.rfmodelpredict:main
+    - rfmodeltrain=mlgenotype.rfmodeltrain:main
+    - rftrainpredict=mlgenotype.rftrainpredict:main
+    - figure1calcs=mlgenotype.figure1calcs:main
+
+requirements:
+  build:
+    - python
+    - pip
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pandas >=1.0
+    - scikit-learn ==1.0.2
```

Comment:
maybe use `>=` to avoid conflict with other packages

Reply:
I was pinning to scikit-learn 1.0.2 because I used it for manuscript results, and when I run new data on fitted models from that version, I get a warning that the model was fitted using a different scikit-learn version.

That being said, the results seem to be the same for more recent versions, so I could just look into suppressing the warning (which also doesn't seem ideal)?